flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogTable.java

```diff
@@ -124,6 +126,25 @@ public void testCreateTable() throws TableNotExistException {
     Assert.assertEquals(Maps.newHashMap(), catalogTable.getOptions());
   }
 
+  @Ignore("Enable this after upgrade flink to 1.12.0, because it starts to support 'CREATE TABLE IF NOT EXISTS")
```

Comment:
Currently, the flink 1.12.x & master branch does not provide `IF EXISTS` or `IF NOT EXISTS` for `RENAME TABLE` and `ALTER TABLE`,  so I did not provide unit tests.